src/js/components/Beacon/BeaconQueryUi.tsx

```diff
@@ -242,8 +248,10 @@ const BeaconQueryUi = () => {
       <Card
         title={td('Search')}
         style={{ borderRadius: '10px', maxWidth: '1200px', width: '100%', ...BOX_SHADOW }}
-        bodyStyle={CARD_BODY_STYLE}
-        headStyle={CARD_HEAD_STYLE}
+        styles={{
```

Comment:
this object could be a constant itself (`CARD_STYLES` ?) in `beaconConstants` or somewhere